main.js

```diff
@@ -0,0 +1,119 @@
+var weather = [];
```

Comment:
Using var is pre-es6 which is fine for this project cause there are no rules against it, but I would definitely encourage use of post es6 conventions of using const and let